sound/soc/sof/intel/hda-stream.c

```diff
@@ -512,6 +495,7 @@ irqreturn_t hda_dsp_stream_threaded_handler(int irq, void *context)
 
 			snd_hdac_stream_writeb(s, SD_STS, SD_INT_MASK);
```

Comment:
Is this really correct? writing to SD_STS clears respective bits, right? If that's the case we shouldn't clear all bits, we should only clear the bits, that we picked up for processing, i.e.
```
snd_hdac_stream_writeb(s, SD_STS, sd_status);
```